docs/src/sway-program-types/libraries.md

```diff
@@ -36,7 +36,7 @@ pub struct MyStruct {
 }
 ```
 
-Libraries are composed of just a `Forc.toml` file and a `src` folder, unlike usual Sway projects which usually contain a `tests` folder and a `Cargo.toml` file as well. An example of a library's `Forc.toml`:
+Libraries are composed of just a `Forc.toml` file and a `src` directory, unlike contract Sway projects which usually contain a `tests` directory and a `Cargo.toml` file as well. An example of a library's `Forc.toml`:
```

Comment:
I know what you mean, but _contract sway projects_ is difficult for me to understand on first pass. Since we said _Libraries_ and not _Library Sway projects_, maybe this can just say _unlike contracts_?
```suggestion
Libraries are composed of just a `Forc.toml` file and a `src` directory, unlike contracts which usually contain a `tests` directory and a `Cargo.toml` file as well. An example of a library's `Forc.toml`:
```